pkgs/development/tools/build-managers/meson/default.nix

```diff
@@ -47,10 +47,10 @@ python3Packages.buildPythonApplication rec {
 
   crossFile = writeTextDir "cross-file.conf" ''
     [binaries]
-    c = '${stdenv.cc.targetPrefix}cc'
-    cpp = '${stdenv.cc.targetPrefix}c++'
-    ar = '${stdenv.cc.bintools.targetPrefix}ar'
-    strip = '${stdenv.cc.bintools.targetPrefix}strip'
+    c = '${stdenv.targetPlatform.config}-cc'
+    cpp = '${stdenv.targetPlatform.config}-c++'
+    ar = '${stdenv.targetPlatform.config}-ar'
+    strip = '${stdenv.targetPlatform.config}-strip'
```

Comment:
Can you use `targetPackages.stdenv...targetPrefix`?

Reply:
I'd just redelegate the ticket to you, because I do not understand the ongoing refactoring, which names are deprecated, which are prospective, etc

Reply:
This code just has been copy-pasted from another nix-file